src/github.com/getlantern/proxiedsites/proxiedsites.go

```diff
@@ -1,269 +1,217 @@
-// package proxiedsites is a module used to manage the list of sites
-// being proxied by Lantern
-// when the list is modified using the Lantern UI, it propagates
-// to the default YAML and PAC file configurations
+// proxiedsites manages a list of proxied sites, including a default set of
+// sites (cloud) and user-applied customizations to that list. It provides an
+// implementation of the http.Handler interface that serves up a PAC file based
+// on the currently active proxied sites (cloud + additions - deletions).
 package proxiedsites
```

Comment:
proxiedsites was simplified considerably.

1. It's all just package level functions now (no ProxiedSites data structure)

2. There's no pac file, it's just bytes in memory. ProxiedSites provides a function that can be used as an HTTP handler that serves up the pac file from memory.

3. Additions and Deletions were factored into a Delta data structure since these are sometimes sent independently of the Global list.

4. It follows the usual Configure() paradigm.  Also, by having Configure() return a Delta representing the changes, we no longer need any channels in this package.